components/tools/OmeroPy/src/omero/plugins/metadata.py

```diff
@@ -0,0 +1,403 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+   Metadata plugin
+
+   Copyright 2015 Glencoe Software, Inc. All rights reserved.
+   Use is subject to license terms supplied in LICENSE.txt
+
+"""
+
+import logging
+import re
+import sys
+
+import omero
+from omero.cli import BaseControl
+from omero.cli import CLI
+from omero.cli import ProxyStringType
+from omero.constants import namespaces
+from omero.gateway import BlitzGateway
+from omero.util import populate_metadata, populate_roi
+
+HELP = """metadata methods
```

Comment:
The first help line looks a bit odd compared to most of our usual first lines. Would something like:

Metadata utitities

be better? (cf fs plugin)